File: app/src/main/java/com/savvasdalkitsis/gameframe/draw/model/AbstractDrawingTool.java
package com.savvasdalkitsis.gameframe.draw.model;

class AbstractDrawingTool implements DrawingTool {

    @Override
    public void drawOn(Layer layer, int startColumn, int startRow, int column, int row, int color) {
    }

    @Override
    public void finishStroke(Layer selectedLayer) {
    }
}

File: app/src/main/java/com/savvasdalkitsis/gameframe/draw/model/AbstractDrawingTool.kt
package com.savvasdalkitsis.gameframe.draw.model

open class AbstractDrawingTool : DrawingTool {

    override fun drawOn(layer: Layer, startColumn: Int, startRow: Int, column: Int, row: Int, color: Int) {}

    override fun finishStroke(selectedLayer: Layer) {}
}
